docs/_source/practical_guides/annotate_dataset.md

```diff
@@ -9,7 +9,7 @@
 
 After pushing a `FeedbackDataset` to Argilla, as explained in [Create a Dataset](create_dataset.md), you can start annotating it through the Argilla UI.
 
-As you open the dataset in the UI, you will see by default the records with `Pending` responses, i.e. records that still don't have a response (annotation), in a single-record view. On the left, you can find the record to annotate and to the right is the form with the questions to answer. You can navigate through the records using the `Prev` and `Next` buttons in the bottom bar. You may use the search bar to filter the records based on specific words or phrases.
+As you open the dataset in the UI, you will see by default the records with `Pending` responses, i.e. records that still don't have a response (annotation), in a single-record view. On the left, you can find the record to annotate and to the right is the form with the questions to answer. You can navigate through the records using the `Prev` and `Next` buttons in the bottom bar. You may use the search bar and filters --as explained in the [🔎 Filter and query datasets page](/practical_guides/filter_dataset.md)-- to make the process faster and more efficient.
```

Comment:
maybe we can also reference the annotation workflows here?

Reply:
The only workflow that applies here is the Semantic Search (the others don't apply to Feedback Datasets) and it's a copy of what is found in one of the sections of the page that I referenced here, so I'll leave it like this for now.